playground/remix-node/app/root.tsx

```diff
@@ -42,6 +42,9 @@ export const meta: MetaFunction = () => ({
 export const CatchBoundary = ClerkCatchBoundary();
 
 function App() {
+  const loaderData = useLoaderData();
+  console.log('root: ', { loaderData });
```

Comment:
i've validated the fix using the log of this line. Before the fix this was logging only the clerk state and after the fix this was logging also an `user` .

Reply:
Great